tests/test_uoi_lasso.py

```diff
@@ -184,3 +185,54 @@ def test_fit_intercept():
     lasso = UoI_Lasso(fit_intercept=False)
     assert not lasso._selection_lm.fit_intercept
     assert not lasso._estimation_lm.fit_intercept
+
```

Comment:
Should add a test (or modify an existing test) to use the new solver in a uoi model.